RecoParticleFlow/PFSimProducer/plugins/SimPFProducer.cc

```diff
@@ -99,6 +101,8 @@ SimPFProducer::SimPFProducer(const edm::ParameterSet& conf)
       neutralEMThreshold_(conf.getParameter<double>("neutralEMThreshold")),
       neutralHADThreshold_(conf.getParameter<double>("neutralHADThreshold")),
       useTiming_(conf.existsAs<edm::InputTag>("trackTimeValueMap")),
+      useTimingQuality_(conf.existsAs<edm::InputTag>("trackTimeQualityMap")),
```

Comment:
Please add a fillDescriptions(...) method for this producer, so that you can get rid of this esistAs check (among other things...)

Reply:
Indeed, if so some modification for the  ```useTiming_``` and ```useTimingQuality_``` bool's should be allowed.
But, by the way: is it even foreseen that such "trackTimeValueMap" and "trackTimeQualityMap" can not exist? Couldn't we give them for granted, and in that case those two bool's are not even needed any more?

Reply:
They don't exist in the "fastsim" implementation with smeared sim times, and there are still eras defined for this (though I'm not sure if they're still actively used anywhere).

Reply:
@bendavid I agree that this PR is not the place where to fix the issue of missing fillDescription in that producer, taking also into account that those functionalities are missing in all producers in RecoParticleFlow/PFSimProducer.

Still, I believe that all producers in that package should profit of the Configuration validation. Issue #29449 was created to keep note of it.